assets/js/vendor/cards-expandable.js

```diff
@@ -0,0 +1,12 @@
+// JS for expandable cards (.va-card--expands)
+$(document).ready(function() {
+	$('.va-card-body').on('click', function(e){
+    	if(e.target.nodeName == 'DIV') {
+    		$(e.currentTarget).parent().toggleClass('va-card--hasdrawer--open');
+    	}
+    	// If the drawer is open, disable links in the card's body
+    	if(e.target.nodeName == 'A' && $(e.currentTarget).parent().hasClass('va-card--hasdrawer--open')) {
```

Comment:
Is this to disable the `h4` in the card when the drawer is open? If so, why doesn't the drawer just cover the whole card if we don't want the header to be clickable anyway?